resources/AssignedAuthor.xml

```diff
@@ -20,32 +19,28 @@
   <kind value="logical"/>
   <abstract value="false"/>
   <type value="AssignedAuthor"/>
-  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Element"/>
+  <baseDefinition value="http://hl7.org/fhir/cda/StructureDefinition/LogicalElement"/>
   <derivation value="specialization"/>
-  <snapshot>
+  <differential>
     <element id="AssignedAuthor">
       <path value="AssignedAuthor"/>
       <min value="1"/>
       <max value="1"/>
-      <base>
-        <path value="AssignedAuthor"/>
-        <min value="1"/>
-        <max value="1"/>
-      </base>
     </element>
-    <element id="AssignedAuthor.nullFlavor">
```

Comment:
I assume you removed nullFlavor because it is in LogicalElement? I can't see/find LogicalElement so I can't check...

Reply:
Yes Nullflavor should not be removed. It was in the snapshot but not in the differential. This is another inconsistency between snapshot and differential :-( will provide an update.